Test Failure: junit.framework.TestSuite.com.ibm.ws.logging.hpel.fat.FATSuite

RTC 275563

```
com.ibm.ws.logging.hpel.fat.FATSuite:java.lang.RuntimeException: java.lang.RuntimeException: Error getting server
at componenttest.custom.junit.runner.FATRunner.injectLibertyServers(FATRunner.java:743)
at componenttest.custom.junit.runner.FATRunner.run(FATRunner.java:166)
Caused by: java.lang.RuntimeException: Error getting server
at componenttest.topology.impl.LibertyServerFactory.getLibertyServer(LibertyServerFactory.java:204)
at componenttest.topology.impl.LibertyServerFactory.getLibertyServer(LibertyServerFactory.java:97)
at componenttest.topology.impl.LibertyServerFactory.getLibertyServer(LibertyServerFactory.java:67)
at componenttest.custom.junit.runner.FATRunner.injectLibertyServers(FATRunner.java:709)
Caused by: componenttest.exception.TopologyException: Unable to delete pre-existing server directory: C:/jazz-build-engines/jazz112.hursley.ibm.com/EBC.PROD.WASRTC/build/dev/image/output/wlp/usr/servers/HpelServer
at componenttest.topology.impl.LibertyServer.setup(LibertyServer.java:647)
at componenttest.topology.impl.LibertyServer.<init>(LibertyServer.java:588)
at componenttest.topology.impl.LibertyServerFactory.getLibertyServer(LibertyServerFactory.java:160)
```

--- a/dev/com.ibm.ws.logging.hpel_fat/fat/src/com/ibm/ws/logging/hpel/fat/CommonTasks.java
+++ b/dev/com.ibm.ws.logging.hpel_fat/fat/src/com/ibm/ws/logging/hpel/fat/CommonTasks.java
@@ -481,7 +481,9 @@ public class CommonTasks {
         }
         Properties bootstrapProps = new Properties();
         bootstrapProps.setProperty("websphere.log.provider", "binaryLogging-1.0");
-        bootstrapProps.store(bootstrapFile.openForWriting(true), null);
+        OutputStream os = bootstrapFile.openForWriting(true);
+        bootstrapProps.store(os, null);
+        os.close();
         return true;
 //        Workspace tempWorkSpace = HpelSetup.getCellUnderTest().getWorkspace();
 //        getUnifiedLoggingService(aServer).getAttributeByName("enable").setValue(enable);
@@ -925,9 +927,13 @@ public class CommonTasks {
         LibertyServer server = appServer;
         RemoteFile bootstrapFile = server.getServerBootstrapPropertiesFile();
         Properties bootstrapProps = new Properties();
-        bootstrapProps.load(bootstrapFile.openForReading());
+        InputStream is = bootstrapFile.openForReading();
+        bootstrapProps.load(is);
         bootstrapProps.remove(propertyName);
-        bootstrapProps.store(bootstrapFile.openForWriting(false), null);
+        is.close();
+        OutputStream os = bootstrapFile.openForWriting(false);
+        bootstrapProps.store(os, null);
+        os.close();
 
     }
 

